engine/src/test/java/org/camunda/bpm/engine/test/api/repository/ProcessDefinitionQueryTest.java

```diff
@@ -108,6 +111,36 @@ public void testQueryByInvalidDeploymentId() {
     repositoryService.createProcessDefinitionQuery().deploymentId(null);
   }
 
+  @Test
+  public void testQueryByDeploymentTimeAfter() {
+    List<Deployment> deployments = repositoryService.createDeploymentQuery().list();
+
+    for (Deployment deployment : deployments) {
+      List<ProcessDefinition> processDefinitions = repositoryService.createProcessDefinitionQuery().deployedAfter(deployment.getDeploymentTime()).list();
+      for (ProcessDefinition processDefinition : processDefinitions) {
+        Deployment singleDeployment = repositoryService.createDeploymentQuery().deploymentId(processDefinition.getDeploymentId()).singleResult();
+        // all results should have a later deployment time than the one used in the query
+        assertThat(singleDeployment.getDeploymentTime()).isAfter(deployment.getDeploymentTime());
+      }
+    }
+  }
+
+  @Test
+  public void testQueryByDeploymentTimeAt() {
```

Comment:
Same here, this test is only works if time elapses between deployments. Please also verify that the returned definitions are actually the ones you expect and not just that the count is correct.